decisioncentral/branch-overrides.yaml

```diff
@@ -10,7 +10,7 @@ modules:
     - name: cct_module
       git:
         url: https://github.com/jboss-openshift/cct_module.git
-        ref: 0.39.0
+        ref: 0.44.0
```

Comment:
needs to be updated on the tag-overrides.yaml as well.

Reply:
Done